System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"
+
+SET filePath="System Information\System_Information.txt"
+
+:: Run SYSTEMINFO
+echo Running command: systeminfo
+echo ------- Output of command: systeminfo --------- > %filePath%
+systeminfo >> %filePath%
+
+:: Run WMIC
+echo Running command: WMIC
+echo ------- Output of command: WMIC --------- >> %filePath%
+wmic /APPEND:%filePath% product get name,vendor,version >> nul
+
+::Run Dxdiag and append data to System_Information.txt
+echo Running command: dxdiag
+echo ------- Output of command: dxdiag --------- >> %filePath%
+dxdiag /T dxdiag.txt
+type dxdiag.txt >> %filePath%
+del dxdiag.txt
+
+::Run PsInfo and append data to System_Information.txt

```

Comment:
The original motivation to use psinfo was this [comment from Han](https://issues.gpii.net/browse/GPII-2199?focusedCommentId=23554&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23554). He was asking to take the input from psinfo /s, which also outputs the list of software, which we're already getting from WMIC's output. I'll ask Han to double check on this, but my understanding is that we're fine with systeminfo, wmic and dxdiag.